packages/react.js

```diff
@@ -7,6 +7,6 @@
  */
 
 import { Client } from '../src/client/react.js';
-import Lobby from '../src/lobby/react.js';
+import Lobby, { LobbyPhases } from '../src/lobby/react.js';
```

Comment:
Is this change really necessary? I would prefer to not expose too many types to the public API (requires documentation and generally adds to the cognitive overhead).

Reply:
I don't think it's really necessary, I'll remove it.